src/main/steps/application-within-proceedings/download-form/content.ts

```diff
@@ -0,0 +1,106 @@
+import { AWPApplicationType } from '../../../app/case/definition';
+import { TranslationFn } from '../../../app/controller/GetController';
+import { FormContent } from '../../../app/form/Form';
+import { getCasePartyType } from '../../../steps/prl-cases/dashboard/utils';
+import { getApplicationDetails } from '../utils';
+
+export const en = {
+  title: 'Download the form',
+  explainRequest: 'You can explain your request by completing and uploading the form',
+  nextSteps: 'Next steps',
+  line1: 'Download the form',
+  from: 'from',
+  govUk: 'GOV.UK (opens in a new tab)',
+  line2: 'Complete the application form',
+  line3: 'Save the application form onto your device',
+  line4: 'Return to upload your application form',
+  warning: 'Warning',
+  warningText: 'You need to return to upload your application form for your request to be completed.',
+  onlyContinue: 'Continue',
+  cancel: 'Cancel',
+};
+
+export const cy: typeof en = {
+  title: 'Download the form (welsh)',
+  explainRequest: 'You can explain your request by completing and uploading the form (welsh)',
+  nextSteps: 'Next steps (welsh)',
+  line1: 'Download the form (welsh)',
+  from: 'from (welsh)',
+  govUk: 'GOV.UK (opens in a new tab) (welsh)',
+  line2: 'Complete the application form (welsh)',
+  line3: 'Save the application form onto your device (welsh)',
+  line4: 'Return to upload your application form (welsh)',
+  warning: 'Warning (welsh)',
+  warningText: 'You need to return to upload your application form for your request to be completed. (welsh)',
+  onlyContinue: 'Parhau',
+  cancel: 'Canslo',
+};
+
+const languages = {
+  en,
+  cy,
+};
+
+const downloadUrls = {
```

Comment:
create a method which will return the downloadform url

Reply:
Moved to config in utils file